### ISSUE TITLE
[BUG] Unable to import signed PDF

### ISSUE THREAD

**Issue by dokuhebi** [jonaswinkler/paperless-ng#575]
**Describe the bug**
PaperlessNG is unable to import signed PDF.

**To Reproduce**
1. Upload signed PDF to consume folder
2. PDF will not import
3. Check logs to see "Error while consuming document doc.pdf: Input PDF is encrypted. The encryption must be removed to perform OCR."

**Expected behavior**
The PDF should be imported.

**Screenshots**

**Webserver logs**
```
2/16/21, 5:53 PM ERROR Error while consuming document [redacted]: Input PDF is encrypted. The encryption must be removed to perform OCR. For information about this PDF's security use qpdf --show-encryption infilename You can remove the encryption using qpdf --decrypt [--password=[password]] infilename

2/16/21, 5:53 PM ERROR Error while consuming document [redacted]: Input PDF is encrypted. The encryption must be removed to perform OCR. For information about this PDF's security use qpdf --show-encryption infilename You can remove the encryption using qpdf --decrypt [--password=[password]] infilename
```

**Relevant information**
 - Running a the paperless-ng container on UnRAID

**Comment by jonaswinkler** [comment]
Two things:

* Please set `PAPERLESS_OCR_MODE=force`, restart, and try again. Please report if this works and if text is correctly added to the content field of paperless. The result will be blurry. This is not a permanent solution, I just need to know if it works.
* Even better, if you come across an encrypted PDF document that does not have any kind of private information (i.e., I can freely push that into the repository as a test file), please upload that here. I don't know how to make these files.

**Comment by rssfed23** [comment]
Here's a signed & encrypted PDF (made with Acrobat Pro and signed via Adobe signed).
Although there are a few different ways people sign and encrypt PDFs I think, this is likely one of the more common methods (default settings across Acrobat and Adobe Sign including a digital "signature")

[signed_encrypted_.pdf](https://github.com/jonaswinkler/paperless-ng/files/6016201/signed_encrypted_.pdf)

**Comment by jonaswinkler** [comment]
Thank you!

**Comment by jonaswinkler** [comment]
I'm also not exactly sure what the actual issue here is. The current 1.1.4 release adds your demo file just fine.

**Comment by dokuhebi** [comment]
I can send the original file that failed to import, but it's got personal information in it, so I'd like it kept out of the repo.  There's nothing private... just my home address.  Is there a good way for me to send it to you to check it out?

**Comment by jonaswinkler** [comment]
@dokuhebi Not sure yet. Are you using any non-default configuration options (regarding ocr)?

**Comment by dokuhebi** [comment]
Everything is default out of the box. (Great tool, btw... it's greatly decreased the arguments between me and my wife.)

I tried dev and it imported fine!  Whatever you did, keep it. :-)

**Comment by jonaswinkler** [comment]
Thanks for testing! Remember to go back to `latest` or `1.1.4`. `dev` is generally unstable.

> UNIQUE constraint failed: documents_document.checksum

The only instance where I've experienced this issue over here is when I tried to add the same document (exactly the same) twice, at the same time. For example, by copying it twice into the consumption directory. If that happens, paperless starts to process both documents in parallel, the first document gets added, but the second one fails (since a document with that checksum already exists). If you remember doing something like that, then that's the reason. If not, this is a different issue and we need to investigate.

Paperless usually detects duplicates by checksum and aborts immediately, but that does not work for this particular case.
